content/applications/general/email_communication/email_domain.rst

```diff
@@ -126,7 +119,14 @@ If the domain name uses :abbr:`DMARC (Domain-based Message Authentication, Repor
 Conformance)` and has defined one of these policies, the domain must be :abbr:`SPF (Sender Policy
 Framework)` compliant or enable :abbr:`DKIM (DomainKeys Identified Mail)`.
 
-.. danger::
+If a partner, customer, or vendor, uses :abbr:`DMARC (Domain-based Message Authentication,
+Reporting, & Conformance)` and has defined one of these policies, the Odoo server cannot relay
+emails from this partner to the database users.
```

Comment:
Not the case anymore with the notifications@ configuration.
As we do not spoof anymore

Reply:
Noted, I will remove this section.